name/static/name/js/map.js

```diff
@@ -18,17 +18,18 @@ $(function() {
 
   // Get the JSON payload of locations from the endpoint.
   $.get(url).done(function(data) {
+    // console.log(data)
```

Comment:
This console.log(data) is commented out and thus should probably be removed. I'm guessing it's left over from debugging?